docs/extend/extend-cli/cli-developing-a-plugin.md

```diff
@@ -2,218 +2,339 @@
 
-![Side by Side Diff](../../images/guides/CLI/htmlDiff.png)
+1. Open a terminal and run the command `mkdir zowe-tutorial`.
+    
+    **Note:** All the files created through this tutorial are saved in this tutorial directory.
+2. Enter `cd zowe-tutorial` to change directory into your `zowe-tutorial` folder.
+3. Download the [source code zip file](https://github.com/zowe/zowe-cli-sample-plugin/archive/refs/heads/master.zip) from the Zowe CLI sample plug-in repository.
+4. In your File Explorer, extract the zip file to the  `zowe-tutorial` folder.
+5. Rename the `zowe-cli-sample-plugin-master` directory to `files-util`.
+    
+    This is the project directory used throughout the rest of this tutorial. 
+6. Delete all content within the following folders:
+    - `src/api`
+    - `src/cli`
+    - `docs` folders
+    - `__tests__/__system__/api`
+    - `__tests__/__system__/cli`
+    -  `__tests__/api`
+    - `__tests__/cli`
+7. Return to your terminal and run `cd files-util` to enter the project directory.
+8. Enter `git init` to set up a new Git repository.
+9. Enter `git add --all` to *stage* (track) all files in the current directory with Git.
 
-Completed source for this tutorial can be found on the `develop-a-plugin` branch of the zowe-cli-sample-plugin repository.
+10. Enter `git commit --message "Initial commit"` to save a snapshot of the staged files in your repository.
+11. Run `npm install` to install third-party dependencies defined in the `package.json` file of your Node.js project.
 
-### Cloning the sample plug-in source
- Clone the sample repo, delete the irrelevant source, and create a brand new plug-in. Follow these steps:
+    When successful, **[WHAT IS THE USER SUPPOSED TO SEE?]**.
 
-1. `cd` into your `zowe-tutorial` folder
-2. `git clone https://github.com/zowe/zowe-cli-sample-plugin files-util`
-3. `cd files-util`
-4. Delete the `.git` (hidden) folder.
-5. Delete all content within the `src/api`, `src/cli`, and `docs` folders.
-6. Delete all content within the `__tests__/__system__/api`, `__tests__/__system__/cli`, `__tests__/api`, and `__tests__/cli` folders
-7. `git init`
-8. `git add .`
-9. `git commit -m "initial"`
+**To create a unique plug-in:** Change the `files-util` directory to a name applicable for your project.
 
-### Changing package.json
-Use a unique `npm` name for your plugin. Change `package.json` name field as follows:
+## Updating `package.json`
+
+Change the name property in the `package.json` file to the plug-in name.
+
+Open the `package.json` file in a text editor and replace the name field with the following information:
 
 ```typescript
   "name": "@zowe/files-util",
 ```
 
-Issue the command `npm install` against the local repository.
+This tutorial uses `@zowe/files-util` as the tutorial plug-in name.
+
+**To create a unique plug-in:** Replace `@zowe/files-util` with a unique plug-in name. This allows you to publish the plug-in under that name to the `npm` registry in the future.
```

Comment:
should we link to npm's doc regarding scopes and/or naming?
- https://docs.npmjs.com/cli/v9/configuring-npm/package-json#name
There is a note at the end of the `name` section that talks about scope.

I guess I want to avoid people thinking they only need to change the `files-util`  part (as recommended in the previous `To create a unique plug-in` note) and keep the `@zowe` scope since it's a plug-in for zowe cli. 
If that's the case, they will not be able to publish their plug-in becuase we (Zowe) owns the rights to the `@zowe` scope.

Reply:
added note on npm scoping.